plf-packaging-resources/src/main/resources/CHANGE_LOG.txt

```diff
@@ -4,768 +4,66 @@ Changelog Platform ${org.exoplatform.platform.version}
-    * [TA-558] - [Loading Tasks] Exception thrown when we load Tasks by keyword
-    * [TA-561] - Log errors when starting eXo platform server that contain exo-tasks Addon
-    * [TA-571] - Filter does not work for the first time
-    * [TA-574] - [Tasks] Warning message when trying to modify task's status
-    * [TA-575] - Reply in Tasks application is displayed with the username of the parent comment owner
-    * [TA-580] - Several problems in editing task description
-    * [TA-582] - [SQLServer] Can't open task detail
-    * [TA-583] - Can't add comments to tasks
-    * [TA-586] - Contraints identifier FK_TASK_COMMENT_PARENT_COMMENT_ID is too long on Oracle database
-    * [TA-587] - Task description field doesn't work
-    * [TA-589] - Task description lost when leaving field with "tab"
-    * [TA-591] - The calendar app is not displayed when assigning a task to a user in the chat room
-    * [TA-595] - [Enterprise-Skin] Wrong Save button position in Project Permission popup
-    * [TA-596] - regression : Links no longer recognized in description and comments
-    * [TA-598] - [Task-Responsive] To assign the task to a person,the input text of Assigned to is hidden
-    * [TA-600] - Unable to add a column status on board view
-    * [TA-602] - Type of calendar is not set with Task Event
-    * [TA-603] - [Security]: XSS executed on task description
-    * [TA-604] - Task duration is wrong when selecting "from time" 12:00
-    * [TA-608] - Placeholder on Tasks description problem
-    * [TA-609] - Tasks created in chat does not contain a link to the task
-    * [TA-610] - No warning message when trying to create a task in chat without due date
-    * [TA-611] - Task Comment button is not enabled when the comment contains only an image
-    * [TA-612] - Select files popup UI is broken
-    * [TA-613] - [TRIBE-262]:Wrong icon for tasks in unified search results
-    * [TA-615] - Task space integration is highly soliciting Space storage
-    * [TA-617] - Nothing appears on Calendar when creating a task using the addon
-    * [TA-619] - in Assign Task popu the last assigned user is displayed by default
+    * [PLF-7921] - Disallow listing the rest API services for unauthenticated users.
+    * [PLF-7986] - Enterprise skin style cannot be overridden with custom modules because of too high priority
+    * [PLF-7989] - Inconsistency between the versioning in activity stream and document application
 
 ** Improvement
-    * [TA-213] - The Group by & Sort by view are changed after adding tasks
-    * [TA-623] - the image is not displayed correctly in the Task menu
-    * [TA-626] - the alignment of the image is incorrect in the menu task -> comment
+    * [PLF-7902] - Restrict WebDAV folder listing only for some paths
 
 ** Task
-    * [TA-517] - Move to re-use the new suggester component
+    * [PLF-7919] - Upgrade to Tomcat 8.5.30
 
-eXo Chat 2.0.0
+eXo Chat ${org.exoplatform.platform.version}
```

Comment:
${org.exoplatform.platform.version} will be replaced by 5.0.1, which is not the right version for the Chat addon.